app/routes/docs.notices.producers.mdx

```diff
@@ -33,8 +33,8 @@ also happy to work with the mission teams to help construct your alerts.
       <ProcessListHeading type="h3">
          Name Your Kafka Topics
       </ProcessListHeading>
-      Names of Kafka topics follow the format <code>gcn.notices.<i>mission</i>.<i>notice_type</i></code>.
-      Pick a prefix for your Kafka topic names, <code><i>mission</i>.\*</code>.
+      The naming convention for Kafka topics follow the format <code>gcn.notices.<i>mission</i>.<i>NoticesType</i></code>. The mission name should be lower case, while the Kafka topics should be in [CamelCase](https://en.wikipedia.org/wiki/Camel_case) format.
+      Pick a prefix for your Kafka topic names, <code><i>mission</i>.\*</code>. 
```

Comment:
We shouldn't link to wikipedia.  Also, what about gcn.notices.mission.instrument.NoticeType?  Instrument is optional. And NoticesType should be NoticeType

Reply:
Ok, other reference sites are worse and have advertisements. "instrument.NoticeType" extra instrument word is my mistake. 
I was trying to keep Notices as a consistent word for GCN Notices.

Reply:
> We shouldn't link to wikipedia.

Why not?

Reply:
It's a NASA policy when it comes to public communications to only link to trusted other agency or institution websites.  Anyone can change the content of wikipedia.

Reply:
> It's a NASA policy when it comes to public communications to only link to trusted other agency or institution websites. Anyone can change the content of wikipedia.

Is there a link to that policy? I haven't heard of it before.